orchestrator/jenkins/orchestrator-pipeline.groovy

```diff
@@ -0,0 +1,82 @@
+pipeline_timeout = 10
+
+pipeline {
+    parameters {
+        string(
+            name: 'GIT_REPO',
+            defaultValue: 'https://github.com/percona/orchestrator',
+            description: 'URL to the Orchestrator repository',
+            trim: true)
+        string(
+            name: 'BRANCH',
+            defaultValue: 'master',
+            description: 'Tag/Branch for the Orchestrator repository',
+            trim: true)
+        string(
+            name: 'PS_TARBALL',
+            defaultValue: 'https://downloads.percona.com/downloads/Percona-Server-8.0/Percona-Server-8.0.26-16/binary/tarball/Percona-Server-8.0.26-16-Linux.x86_64.glibc2.12-minimal.tar.gz',
+            description: 'PS tarball to be used for testing',
+            trim: true)
+        string(
+            name: 'CI_ENV_GIT_REPO',
+            defaultValue: 'https://github.com/percona/orchestrator-ci-env.git',
+            description: 'URL to the Orchestrator CI repository',
+            trim: true)
+        string(
+            name: 'CI_ENV_BRANCH',
+            defaultValue: 'master',
+            description: 'Tag/Branch for the Orchestrator CI repository',
+            trim: true)
+    }
+    agent {
+        label 'micro-amazon'
+    }
+    options {
+        skipDefaultCheckout()
+        skipStagesAfterUnstable()
+        timeout(time: 6, unit: 'DAYS')
+        buildDiscarder(logRotator(numToKeepStr: '200', artifactNumToKeepStr: '200'))
+    }
+    stages {
+        stage('Integration Tests') {
+            agent { label 'docker-32gb' }
+            steps {
+                git branch: 'DISTMYSQL-140', url: 'https://github.com/kamil-holubicki/jenkins-pipelines'
```

Comment:
Am I right that once this tested and verified, the links will point to some of our repos ?

Reply:
Yes, as it works as expected, I just committed corrected links.